env-config.yaml

```diff
@@ -0,0 +1,75 @@
+ClusterConfig:
```

Comment:
nit: to not be confused with environment.yaml, i think it's better to rename this file to `config.yaml`